hapi/hedera-protobufs/services/state/network/network.proto

```diff
@@ -52,33 +63,53 @@ message Network {
   repeated NodeMetadata node_metadata = 1;
 
   /**
-   * If set, a list of TSS messages that have been generated by the nodes in
-   * a target network that is receiving a "transplant state". This lets us
-   * preserve the ledger id of the target network even if the transplant
-   * state has no keys (or even roster entries) in common with the target.
-   * This must be exactly the ordered list of TSS messages used to generate
+   * A list of TSS messages that have been generated by the nodes in
+   * a target network that is receiving a "transplant state". <br/>
+   * This gives the capability to preserve the ledger id of the target network
+   * even if the transplant state has no keys or roster entries in common with
+   * the target.
+   * <p>
+   * This MUST be exactly the ordered list of TSS messages used to generate
    * the target network's ledger id; and must generate the same ledger id
    * given below.
    */
   repeated com.hedera.hapi.services.auxiliary.tss.TssMessageTransactionBody tss_messages = 2;
 
   /**
-   * If set, the expected ledger id of the network.
+   * The ledger id of the network.<br/>
+   * This is the public key that identifies the ledger and can be used by the
+   * network to create a Block Proof or ledger signature on a block root hash.
    */
   bytes ledger_id = 3;
 }
 
 /**
- * The full information needed for a node.
+ * The full information needed for a single node in the network state.
  */
 message NodeMetadata {
   /**
-   * The metadata of the node.
+   * A consensus node identifier.
+   * <p>
+   * Node identifiers SHALL be unique _within_ a ledger, and may not be repeated
+   * _between_ shards and realms.<br/>
+   * This value MUST be set.<br/>
+   * This value MUST be a valid single address book node in network state.
    */
-  com.hedera.hapi.node.state.addressbook.Node node = 1;
+  uint64 node_id = 1;
 
   /**
-   * If set, the public part of a prescribed TSS encryption key for the node.
-   */
+    * An elliptic curve public encryption key.<br/>
+    * If set, the public part of a prescribed TSS encryption key for the node.
+    * This is currently an ALT_BN128 curve, but the elliptic curve
+    * type may change in the future. For example,
+    * if the Ethereum ecosystem creates precompiles for BLS12_381,
+    * we may switch to that curve.
+    * <p>
+    * This value SHALL be specified according to EIP-196 and EIP-197 standards,
+    * See <a href='https://eips.ethereum.org/EIPS/eip-196#encoding'>EIP-196</a> and
+    * <a href='https://eips.ethereum.org/EIPS/eip-197#encoding'>EIP-197</a><br/>
```

Comment:
I believe this has changed.  TSS library team is saying their keys are raw binary as used by the arkworks rust code and not encoded according to the EIP standards.